src/prime_rl/trainer/rl/config.py

```diff
@@ -62,6 +63,7 @@ class NCCLWeightBroadcastConfig(BaseModel):
     """Configures the NCCL broadcast."""
 
     type: Literal["nccl"] = "nccl"
+    adapter_only: Annotated[bool, Field(description="Whether to save LoRA adapters only for weight broadcast.")] = False
```

Comment:
@mikasenghaas probably will want to do a base config for the shared field for broadcast, I don't mind keeping it like this for now tho